Liderbord/liderbord
# Create liderbord UI

| **Notion Task**: [Create Liderbord UI Page](https://www.notion.so/matthieu-roux/Create-Liderbord-Page-54831b65477c45079d08c8c20901c042)| 
|:---|

 ## Description
 
 

The goal of this PR is:
- to update the create liderbord UI page so that it looks more like the docs in the figma.
- Parse the tags data into a list of strings.

It will not connect the page to the backend.

## Testing instructions

1. Log in/ Register
2. Press the create liderbords button
3. You can now play around with the new page, and see the console to see the parsed data when clicking submit
4. Please see how you can have errors if the fields are not considered as valid.
5. The design is not perfect... yet... the errors have a white background, and I would like to add the character count, but we will do that later if there is the time. 



## Checklist

### Submission checklist:

1. [x] Does your submission pass tests / work?
7. [x] Have you formatted your code locally before submission? 
8. [x] Have you rebased your changes?

### Reviewer Checklist

1. [ ] I ran and tested the changes on my machine.
2. [ ] I read the code changes in detail.
